ql/src/test/queries/clientpositive/create_table.q

```diff
@@ -0,0 +1,39 @@
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.create.as.external.legacy=true;
+
+-- When hive.create.as.external.legacy is true, the tables created with
+-- 'managed' or 'transactional' are ACID tables but the tables create
+-- without 'managed' and 'transactional' are non-ACID tables.
+-- Note: managed non-ACID tables are allowed because tables are not
+-- transformed when hive.in.test is true.
+
+-- Create tables with 'transactional'. These tables have table property
+-- 'transactional'='true'
+create transactional table test11 as select 1;
+show create table test11;
+describe formatted test11;
+
+create transactional table test12 as select 1;
```

Comment:
why are all the test cases duplicated here?

Reply:
I'll change the test cases then rebase and submit again :)